cmd/config-translator/translator.go

```diff
@@ -32,6 +32,8 @@ func initFlags() {
 	var inputMode = flag.String("mode", "ec2", "Please provide the mode, i.e. ec2, onPrem")
 	var inputConfig = flag.String("config", "", "Please provide the common-config file")
 	var multiConfig = flag.String("multi-config", "remove", "valid values: default, append, remove")
+	var strictValidation = flag.Bool("strict-validation", false, "Return exit code 99 if validating agent's json config fails. Valid values: true, false")
```

Comment:
One thing I thought about that I think we need to test just so we know how this `bool` flag behaves.

We would want to confirm how to pass in the flag on different OSes, namely *NIX (Linux, Unix) versus Windows.

Reply:
Yes. I have tested on different platforms. Only needs to pass with as `--strict-validation`, for example: `./config-translator --config ./agent-config.json --strict-validation` without passing any values(if passing, should be with --strict-validation=true)